sdk/ai/azure-ai-voicelive/src/test/java/com/azure/ai/voicelive/VoiceLiveSessionOptionsTest.java

```diff
@@ -66,13 +67,15 @@ void testSetInstructionsWithNull() {
     void testSetAndGetVoice() {
         // Arrange
```

Comment:
Also noticing that the tests dont follow https://github.com/Azure/azure-sdk-for-java/wiki/Parameterized-test-for-SDK-live-test or recording framework support https://github.com/Azure/azure-sdk-for-java/wiki/Test-Proxy-Migration is this intentional?